nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters
# Tags with posts add only

# Description
Developed two features:

 1)  feature will allow you to add a tag directly to your post and will display them in the details view


# Ticket Reference
> [#17  - Add a Tag to a Post] (https://github.com/nss-day-cohort-41/tabloid-fullstack-crop-circle-cavorters/issues/17)


## Type of change


# Testing Instructions
- Run the API in Visual studio, and `npm serve` the client folder.
- Type `localhost:3000/users` into the browser bar.
- Start program with Tabloid
- Go to localhost:5001 in your browser.
-Login with "admin@example.com"
- Navigate to the `Posts` link in the navbar.
- You will see the list of posts, single out a post your interested in and press the "View details"
- Now press the add tag button/image under your posts title
-You should be led to a list of current tags to select from, choose "1" and submit
-The user will be redirected to the details view of the post 
"Your postTag will be shown" under the subject


# Checklist:
